atlasdb-client/src/test/java/com/palantir/atlasdb/persistent/rocksdb/RocksDbPhysicalPersistentStoreTests.java

```diff
@@ -115,21 +119,21 @@ public void droppingTwoTimesFailsOnSecond() {
     public void testMultiPut() {
         timestampMappingStore.multiPut(
                 defaultNamespace,
-                ImmutableMap.of(1L, 2L, 3L, 4L));
+                ImmutableMap.of(KEY, VALUE, KEY2, VALUE2));
 
-        assertThat(timestampMappingStore.get(defaultNamespace, 1L)).isEqualTo(2L);
-        assertThat(timestampMappingStore.get(defaultNamespace, 3L)).isEqualTo(4L);
+        assertThat(timestampMappingStore.get(defaultNamespace, KEY)).hasValue(VALUE);
+        assertThat(timestampMappingStore.get(defaultNamespace, KEY2)).hasValue(VALUE2);
     }
 
     @Test
     public void testMultiGet() {
-        timestampMappingStore.put(defaultNamespace, 1L, 2L);
-        timestampMappingStore.put(defaultNamespace, 3L, 4L);
+        timestampMappingStore.put(defaultNamespace, KEY, VALUE);
+        timestampMappingStore.put(defaultNamespace, KEY2, VALUE2);
 
-        assertThat(timestampMappingStore.multiGet(defaultNamespace, ImmutableList.of(1L, 2L, 3L)))
+        assertThat(timestampMappingStore.multiGet(defaultNamespace, ImmutableList.of(KEY, KEY2, "random".getBytes())))
                 .containsExactlyInAnyOrderEntriesOf(ImmutableMap.of(
-                        1L, 2L,
-                        3L, 4L)
+                        KEY, VALUE,
+                        KEY2, VALUE2)
                 );
```

Comment:
This assertion feels fragile. In particular if I change the assertion to

```
        assertThat(timestampMappingStore.multiGet(defaultNamespace, ImmutableList.of(KEY, KEY2, "random".getBytes())))
                .containsExactlyInAnyOrderEntriesOf(ImmutableMap.of(
                        KEY, VALUE,
                        "key2".getBytes(), VALUE2)
                );
```

it fails. In general I think we want to either use ByteString for the return type of the persistent store, or use sorted maps.